client/src/components/ReviewListEntry.jsx

```diff
@@ -1,31 +1,97 @@
 import React from 'react';
+import ImagesList from './ImagesList.jsx';
+import styled from 'styled-components'
 const moment = require('moment');
-moment().format();
 
-const ReviewListEntry = (props) => {
+class ReviewListEntry extends React.Component {
+    constructor(props) {
+       super(props);
+       this.state = {
+          process: false,
+       }
+       this.updateHelpfulVote = this.updateHelpfulVote.bind(this);
+    }
+    
+    updateHelpfulVote(data) {
+      let url;
+      if(this.state.process) {
+         url =  `/${data.loc_id}/${data.review_id}/1`;
+      } else {
+         url =  `/${data.loc_id}/${data.review_id}/0`;   
+      }
+
+      fetch(url,{
+        method: "PATCH",
+      })
+      .then(() => {
+         this.props.reviews.reload();
+      })
+      this.setState({process: !this.state.process});
+    }
+    renderReviews(props) {
+      return this.props.reviews.data.map((review, idx) => {
+        const name = review.name.split(' ');
+        const nameAndInitial = `${name[0]} ${name[1][0]}.`; 
+       
+        return (
+          <Container key = {idx}>
+            <img src ={review.avatar} alt="Smiley face" height="70" width="70"></img>
+            <span><b>{nameAndInitial} </b>recommended this listing</span><span>{moment(review.updatedAt).format('MMMM Do YYYY')}</span>
```

Comment:
Looks like there are some spacing issues here. Possibly just from the view on Github, but can you check to make sure that like 39 and 40 are spaced properly?